cura/Machines/Models/MachineListModel.py

```diff
@@ -98,7 +125,9 @@ def addItem(self, container_stack: ContainerStack, machine_count: int = 0) -> No
         for connection_type in [ConnectionType.NetworkConnection.value, ConnectionType.CloudConnection.value]:
             has_connection |= connection_type in container_stack.configuredConnectionTypes
 
-        self.appendItem({"name": container_stack.getName(),
+        self.appendItem({
+                        "listType": "MACHINE",
```

Comment:
Indentation here (`listType`, `name`) is one space less than the ones that already existed (`id`, etc.).